### ISSUE TITLE
eml_write or write_eml?

### ISSUE THREAD

**Issue by mairindeith** [BIOL548O/Discussion#24]
I'm looking to create EML metadata for my own dataset, and have encountered a few issues with the EML library. The main problem I'm currently facing down is this: 

Although the example posted on the homework page uses the command `eml_write(...)`, R doesn't recognize `eml_write` as a command. Instead, the command listed in the help pages is `write_eml(...)`. When I use this command, however, both the example XML and the file I create from my own data have a parsing errors and can't be viewed. 

Any thoughts on what might be going on?

**Comment by santiagodr** [comment]
you are right!, the command is `write_eml`, I used the following in the example in class, and it created the file
`write_eml(dat, 
          col.defs = col.defs, 
          unit.defs = unit.defs, 
          creator = "Carl Boettiger <cboettig@ropensci.org>", 
          file = "EML_example.xml")`

**Comment by aammd** [comment]
Whoa! thanks guys, I'll change the notes!
